From a4dc82d0541d9387f1ae4f98dd1ee54290ccbfde Mon Sep 17 00:00:00 2001
From: Sandeep Patil <sspatil@google.com>
Date: Wed, 7 Feb 2018 02:56:35 +0000
Subject: [PATCH 104/306] Revert "STOPSHIP: Add UidCpuPower global setting"

This reverts commit 495665dc2f1f3cebc94d5f47629b018634a74074.

Reason for revert: b/73018322
Bug: 73018322

Change-Id: I1b83c4d916159e6961ec30576c7629e57978a72d
(cherry picked from commit a51c496153103a8330f9aa8cb5a670d46bcdec85)
---
 core/java/android/provider/Settings.java               | 10 ----------
 .../src/android/provider/SettingsBackupTest.java       |  1 -
 .../server/am/GlobalSettingsToPropertiesMapper.java    |  1 -
 3 files changed, 12 deletions(-)

diff --git a/core/java/android/provider/Settings.java b/core/java/android/provider/Settings.java
index 46a97e4d8be..e709c532200 100644
--- a/core/java/android/provider/Settings.java
+++ b/core/java/android/provider/Settings.java
@@ -10314,16 +10314,6 @@ public final class Settings {
         */
         public static final String SYS_VDSO = "sys_vdso";
 
-        /**
-        * Uid CpuPower global setting. This links the uid.cpupower system property.
-        * The following values are supported:
-        * 0 -> /proc/uid_cpupower/* are disabled
-        * 1 -> /proc/uid_cpupower/* are enabled
-        * Any other value defaults to enabled.
-        * @hide
-        */
-        public static final String UID_CPUPOWER = "uid_cpupower";
-
         /**
          * An integer to reduce the FPS by this factor. Only for experiments. Need to reboot the
          * device for this setting to take full effect.
diff --git a/core/tests/coretests/src/android/provider/SettingsBackupTest.java b/core/tests/coretests/src/android/provider/SettingsBackupTest.java
index d462e5eac60..32d7dddc13b 100644
--- a/core/tests/coretests/src/android/provider/SettingsBackupTest.java
+++ b/core/tests/coretests/src/android/provider/SettingsBackupTest.java
@@ -389,7 +389,6 @@ public class SettingsBackupTest {
                     Settings.Global.TZINFO_UPDATE_METADATA_URL,
                     Settings.Global.INSTALLED_INSTANT_APP_MIN_CACHE_PERIOD,
                     Settings.Global.INSTALLED_INSTANT_APP_MAX_CACHE_PERIOD,
-                    Settings.Global.UID_CPUPOWER,
                     Settings.Global.UNINSTALLED_INSTANT_APP_MIN_CACHE_PERIOD,
                     Settings.Global.UNINSTALLED_INSTANT_APP_MAX_CACHE_PERIOD,
                     Settings.Global.UNUSED_STATIC_SHARED_LIB_MIN_CACHE_PERIOD,
diff --git a/services/core/java/com/android/server/am/GlobalSettingsToPropertiesMapper.java b/services/core/java/com/android/server/am/GlobalSettingsToPropertiesMapper.java
index fe659784732..9033b55f0b4 100644
--- a/services/core/java/com/android/server/am/GlobalSettingsToPropertiesMapper.java
+++ b/services/core/java/com/android/server/am/GlobalSettingsToPropertiesMapper.java
@@ -41,7 +41,6 @@ class GlobalSettingsToPropertiesMapper {
     //  {Settings.Global.SETTING_NAME, "system_property_name"},
         {Settings.Global.SYS_VDSO, "sys.vdso"},
         {Settings.Global.FPS_DEVISOR, ThreadedRenderer.DEBUG_FPS_DIVISOR},
-        {Settings.Global.UID_CPUPOWER, "uid.cpupower"},
         {Settings.Global.DISPLAY_PANEL_LPM, "sys.display_panel_lpm"},
     };
 
-- 
2.17.1

